index.bs

```diff
@@ -1243,40 +1328,37 @@ Create a constant {{MLOperand}} that can be used in {{MLGraphBuilder}} methods.
     1. Let |operand| be the result of invoking the <a>create MLOperand</a> steps with [=this=], `"constant"` and |descriptor|.
         1. If that throws, re-throw the exception and stop.
     1. Let |bytes| be the result of invoking [[=get a copy of the bytes held by the buffer source=]] given |bufferView|.
-    1. Make a request to the underlying platform to register |operand| as a constant with |bytes| as value and store a reference to the corresponding [=implementation-defined=] object to |operand|.{{MLOperand/[[operand]]}}.
+    1. Make a request to the underlying platform to register |operand| as a tensor constant with |bytes| as value and store a reference to the corresponding [=implementation-defined=] object to |operand|.{{MLOperand/[[operand]]}}.
         1. If that fails, throw an "{{OperationError}}" {{DOMException}} and stop.
     1. Return |operand|.
 </div>
 
 #### The {{MLGraphBuilder/constant(value, type)}} method  #### {#api-mlgraphbuilder-constant-value-type}
+
 <div class="note">
     **Arguments:**
         - *value*: a number
         - *type*: an optional {{MLOperandType}}, by default *"float32"*.
     **Returns:**: an {{MLOperand}} object.
 </div>
-<div algorithm=constant-value-type>
-    The {{MLGraphBuilder/constant(value, type)}} steps are:
+
+The {{MLGraphBuilder/constant(value, type)}} steps are:
+<div algorithm=constant-value-type class=algorithm-steps>
     <div class="note">
         The permissions and context validity have been checked by [[#api-mlgraphbuilder-constructor]] steps.
     </div>
-    1. Let |value| be the first argument.
-        1. If |value| is an array, then let |shape| be the result of invoking [=MLOperand/shape=] given |value|.
-            1. If that throws, re-throw the exception and stop.
-        1. Else if |value| is not a number, then throw a "{{TypeError}}" {{DOMException}} and stop.
-    1. Let |type| be the second argument.
-        1. If |type| is `undefined`, let |type| be `"float32"`.
-        1. Otherwise, if |type| is not one of {{MLOperandType}}, then  throw a "{{TypeError}}" {{DOMException}} and stop.
+    1. If |value| is a [=number=] array, then then throw a "{{TypeError}}" {{DOMException}} and stop.
```

Comment:
My last [comment](https://github.com/webmachinelearning/webnn/pull/365#discussion_r1152847950) may confuse. But according to the current IDL, the check should be if `|value|` is not a number, it throw an error.

Reply:
Sorry, that was a typo, or merge hickup. Fixed :)